apis/v1alpha1/instrumentation_webhook.go

```diff
@@ -236,6 +236,14 @@ func (w InstrumentationWebhook) validate(r *Instrumentation) (admission.Warnings
 	default:
 		return warnings, fmt.Errorf("spec.sampler.type is not valid: %s", r.Spec.Sampler.Type)
 	}
+
+	if r.Spec.Exporter.TLS != nil {
```

Comment:
let's also add a validation that if the scheme for the endpoint contains `https` TLS must be defined.

Reply:
no, this can break already existing installations. Users could configure env vars in the `env` section or directly on their deployments.

Reply:
let's go the reverse and if TLS is set we should block when https is not set. If HTTPS is set, TLS doesn't need to be set.